components/src/molecules/collapse/Collapse.html

```diff
@@ -0,0 +1,24 @@
+<div class="a-collapse">
+  <button
```

Comment:
I think this entire button should be inside scoped slot (with @click event and `collapseLocal` passed through scope).
Then you can remove `titleClass` and move title to prop instead of using slot - for cases where button slot won't be used